percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.25-25.12.rst

```diff
@@ -0,0 +1,69 @@
+
+ |Percona XtraDB Cluster| has temporarily disabled savepoints in triggers and stored functions. The reason is that even having fixed bug :bug:`1438990` and bug :bug:`1464468` we have found more cases where savepoints in triggers break binary logging and replication, resulting in server crashes and broken slaves. This feature will be disabled until the above issues are properly resolved.
+
+Bugs fixed 
+==========
+
+ ``SHOW STATUS LIKE ...`` and ``SHOW STATUS`` were taking time proportional to size of :variable:`gcache.size`. Bug fixed :bug:`1462674`.
+
+ When disk space would get filled with :variable:`gcache.page` files, Galera would crash when the next page file was created. Bug fixed :bug:`1488535`.
+
+ XtraBackup SST didn't clobber :file:`backup-my.cnf` which caused SST to fail. Bug fixed :bug:`1431101`.
+
+ Error from ``process::wait`` was not checked in joiner thread leading to joiner starting erroneously even when SST had failed. Bug fixed :bug:`1402166`.
+
+ Fixes for bug :bug:`1407770` and bug :bug:`1426378` implemented in |Percona XtraDB Cluster| :rn:`5.6.24-25.11` introduced an regression. Update of the :variable:`wsrep_cluster_address` variable, following the update of :variable:`wsrep_provider_options` variable would cause the server deadlocked. Bug fixed PXC-421.
+
+ ``mysqldump`` SST could stall due to a regression in desync mutex introduced in |Percona XtraDB Cluster| :rn:`5.6.24-25.11` by fixing the bug :bug:`1288528`. Bug fixed PXC-423.
+
+ ``mysql_tzinfo_to_sql`` sets :variable:`wsrep_replicate_myisam` variable at session scope so that statements are replicated correctly. Bug fixed PXC-332.
+
+ ``Percona-XtraDB-Cluster-devel-56`` package was not included in the ``Percona-XtraDB-Cluster-full-56`` metapackage on *CentOS* due to a conflict with upstream ``mysql`` package. Bug fixed PXC-381.
+
+ Running ``service mysql start`` and then ``service mysql@boostrap start`` afterwards would cause server shutdown. Bug fixed PXC-385.
+
+ ``NO_WRITE_TO_BINLOG`` / ``LOCAL`` for ``ANALYZE TABLE``, ``OPTIMIZE TABLE``, ``REPAIR TABLE``, ``FLUSH`` commands will ensure it is not written to binary log (as in mysql async replication) and not replicated in wsrep. Bug fixed PXC-391. 
+
+ ``FLUSH LOCAL TABLES WITH READ LOCK`` failure (with non-existent tables) didn't resume the galera provider, causing deadlock. Bug fixed PXC-399.
```

Comment:
As mentioned in https://github.com/percona/documentation/pull/16#discussion-diff-39931727,  saying FLUSH TABLES WITH READ LOCK (FTWRL) should be fine.